cmd/ingress-operator/render.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/cluster-ingress-operator/pkg/manifests"
+)
+
+func render(dir string, prefix string) error {
+	files := []string{
+		manifests.CustomResourceDefinitionManifest,
+		manifests.NamespaceManifest,
```

Comment:
Could we also write out a default ingresscontroller resource so the user does not need to write that from scratch?

Reply:
Can we have a separate followup discussion about the default handling in that regard? It's not required for this part of the implementation and could have negative consequences to consider.